ambry-test-utils/src/main/java/com/github/ambry/quota/ServeEnforcementRecommendation.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright 2020 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.quota;
+
+public class ServeEnforcementRecommendation implements EnforcementRecommendation {
```

Comment:
What is Serve Enforcement?  ShouldThrottle is true but status is 200?

Reply:
Fixed this. shouldThrottle should have returned false.